src/middlewared/middlewared/pytest/unit/utils/test_groupmap.py

```diff
@@ -107,62 +107,56 @@ def test__insert_groupmap(groupmap_dir, local_sid):
 def test__insert_group_membership(groupmap_dir, local_sid):
     entries = [
         SMBGroupMembership(
-            sid='S-1-5-32-544',
-            members=(f'{local_sid}-2000010', f'{local_sid}-2000011')
+            sid=f'{local_sid}-2000010',
+            members=('S-1-5-32-544',)
```

Comment:
Maybe add a few comments through this function explaining what you're trying to test.